virtcontainers/qemu_ppc64le.go

```diff
@@ -7,6 +7,7 @@ package virtcontainers
 
 import (
 	"time"
+        "fmt"
```

Comment:
did you run "go fmt" on this file?

Reply:
Thanks @devimc. Running `go gmt` reordered the changes and it looks proper now